### PR TITLE
A class `@when` shouldn't overrides individual method `@when`s. 

### LINKED ISSUE #4350: A class `@when` overrides individual method `@when`s.
Related https://github.com/wp-cli/wp-cli/pull/4349

If a command class has an `@when`, then any individual method `@when`'s are ignored. In particular if the class has `@when before_wp_load`, then a method with `@when after_wp_load` will be called before WP loads.

### PR DESCRIPTION
**Pull request by miya0001**
Fixes #4350

### CODE REVIEW

**Comment by miya0001** [comment]
I removed `@when after_wp_load` form `wp cli has_command`, because it is executed at `before_wp_loaded` and this command doesn't need WP.
Also, this command is tested with [empty directory](https://github.com/wp-cli/wp-cli/blob/master/features/cli.feature#L4).

**Comment by miya0001** [comment]
I changed my mind. 😊 
I reverted `after_wp_load` for `wp cli has_command` and updated the scenario for this.

File: features/command.feature [hunk 1]
```diff
@@ -1180,3 +1180,125 @@ Feature: WP-CLI Commands
...
+      <?php
+      /**
+       * @when before_wp_load
+       */
+      class Custom_Command_Class extends WP_CLI_Command {
+          /**
+           * @when after_wp_load
+           */
+          public function home_url() {
+             WP_CLI::success( home_url() );
```
Comment (danielbachhuber) [inline features/command.feature]: To make these tests more precise, could we have each method `var_dump( function_exists( 'home_url' ) )`?

With this approach, we'll know definitely whether WordPress has loaded or not.

**Comment by miya0001** [comment]
@danielbachhuber 

> var_dump( function_exists( 'home_url' ) );

It is nice idea. I did it.
